.scripts/dip721/set-controllers.sh

```diff
@@ -1,14 +1,16 @@
 #!/bin/bash
 
-cd $(dirname $BASH_SOURCE) || exit 1
+cd "$(dirname $BASH_SOURCE)" || exit 1
 
 cd ../../ || exit 1
 
 # Args
-CONTROLLER_MAIN=$1
-DIP721_TOKEN_CONTRACT_ID=$2
+NETWORK=$1
+CONTROLLER_MAIN=$2
+DIP721_TOKEN_CONTRACT_ID=$3
 
 dfx canister --no-wallet \
```

Comment:
Is there any advantage to calling this over
```sh
dfx canister --no-wallet --network $NETWORK update-settings $DIP721_TOKEN_CONTRACT_ID --controller $CONTROLLER_MAIN --controller $DIP721_TOKEN_CONTRACT_ID
```

Reply:
@ozwaldorf In some cases is all about how familiar the flag is to the end-user, as before we're only able to set a single controller. I personally don't recall using X number of controller's. As long both communicate intent easily, there shouldn't be any advantage apart from personal syntax choice.

Reply:
https://smartcontracts.org/docs/release-notes/0.8.2-rn.html#_new_feature_dfx_canister_create_now_accepts_multiple_instances_of_controller_argument See release notes here